lib/kdump_utils.pm

```diff
@@ -73,8 +73,7 @@ sub prepare_for_kdump {
         return;
     }
 
-    if (get_var('REPO_0_DEBUGINFO')) {
-        my $snapshot_debuginfo_repo = get_var('REPO_0_DEBUGINFO');
+    if (my $snapshot_debuginfo_repo = get_var('REPO_OSS_DEBUGINFO')) {
```

Comment:
does it make sense to use a fallback to the old variable still? E.g. `get_var('REPO_OSS_DEBUGINFO', get_var('REPO_0_DEBUGINFO'))` ?

Reply:
I don't think so, as this variable was only used for openSUSE and we have changed it in rsync.pl. So the only scenario where it makes sense if someone clones old job and still has REPO_0_DEBUGINFO set, which we can ignore as repos will get cleaned up anyway. But good that you've asked =)